setup.py

```diff
@@ -35,9 +35,9 @@ def read(*parts):
 
 
 install_requires = [
-    'django>=1.8,<2.0',
+    'django>=1.11,<3',
     'babel>=1.3',
-    'django-babel>=0.5.1',
+    'enmerkar==0.7.0',
```

Comment:
Removed django-babel and used enmerkar as it is well maintained as compared to django-babel. Hard coding version as newer version drops support for django<2.1.